launchcontainer/launchcontainer.py

```diff
@@ -298,7 +298,8 @@ def studio_submit(self, data, suffix=''):
             self.project = data['project'].strip()
             self.project_friendly = data['project_friendly'].strip()
             self.project_token = data['project_token'].strip()
-            self.support_email = data['support_email'].strip()
+            self.support_email = None if not data['support_email']\
+                else data['support_email'].strip()
```

Comment:
`data['support_email']` will cause a `KeyError` if `support_email` is not in the dict. You can use
```suggestion
        self.support_email = data.get('support_email', '').strip() or None
```

Reply:
@ahmedaljazzar 
I was avoiding `AttributeError: 'NoneType' object has no attribute 'strip'` here because we are setting the default value for `support_email` to None.

However, I actually think I can leave this as the original code (`self.support_email = data['support_email'].strip()`) because we are converting `None` to '' [here](https://github.com/appsembler/xblock-launchcontainer/blob/master/launchcontainer/launchcontainer.py#L260-L264)

Reply:
@estherjsuh You took a nice step forward here, but technically we still have the same issue with the new change. As python tries to evaluate the condition on empty data (`data={}`):
```python
 not data['support_email']
```
will through that `AttributeError` again.

The change that I suggested achieves the end goal outlined; Python will do the following:
- Try to get `support_email` from data
- If no key found it will return `''`
- Strip the fetched value. This will result to either `email@appsembler.com` or `''` in case fetched `support_email` is `None` or empty.
- If the string we received is empty, we set `self.support_email` to None (That's what `or` does here). 

So no risk of `AttributeError` again.


Here's a little snippet, try to uncomment these lines to see it work in action

```python

data = {
    # 'support_email': ' email@appsembler.com    '
}

class A(object):
    def __init__(self):
        # self.support_email = None if not data['support_email'] else data['support_email'].strip()
        self.support_email = data.get('support_email', '').strip() or None


if __name__ == '__main__':
    a = A()
    print(a.support_email)
```

Reply:
Applying Ahmed's suggestion:

```suggestion
            self.support_email = data.get('support_email', '').strip() or None
```